xdsl/interpreters/pdl.py

```diff
@@ -61,7 +61,7 @@ def match_operand(
         self, ssa_val: SSAValue, pdl_op: pdl.OperandOp, xdsl_val: SSAValue
     ):
         if ssa_val in self.matching_context:
-            return True
```

Comment:
I'm not sure why it was implemented in this way before, but it doesn't make sense to me today, am I missing something?